__init__.py

```diff
@@ -245,11 +240,10 @@ def handle_switch_intent(self, message):
             self.speak_dialog('homeassistant.error.sorry')
             return
 
-    @intent_file_handler('set.light.brightness.intent')
-    def handle_light_set_intent(self, message):
+    def handle_light_set(self, message):
```

Comment:
Same comment, should we make  this method private?

Reply:
can be - will change

Reply:
done